atomisticparsers/lammps/parser.py

```diff
@@ -648,33 +654,52 @@ def get_time_step(self):
         return time_step
 
     def parse_thermodynamic_data(self):
+        sec_run = self.archive.run[-1]
+        sec_system = sec_run.system
+
         time_step = self.get_time_step()
         thermo_data = self.log_parser.get_thermodynamic_data()
         if thermo_data is None:
             thermo_data = self.aux_log_parser.get_thermodynamic_data()
         if not thermo_data:
             return
 
-        sec_run = self.archive.run[-1]
-        sec_sccs = sec_run.calculation
-
         n_thermo = len(thermo_data.get('Step', []))
-        create_scc = True
-        if sec_sccs:
-            if len(sec_sccs) != n_thermo:
-                self.logger.warning(
-                    '''Mismatch in number of calculations and number of property
-                    evaluations!, will create new sections''',
-                    data=dict(n_calculations=len(sec_sccs), n_evaluations=n_thermo))
+        system_steps = [frame.step for frame in sec_system]
```

Comment:
My comments are the same as for gromacs. We can perhaps make a super class for MD parsers. which has only this parse_thermodynamic data

Reply:
I opened an issue for this.